tools/github-labels/repositories.txt

```diff
@@ -0,0 +1,21 @@
+Azure\azure-sdk-for-net
+Azure\azure-sdk-for-net-pr
+Azure\azure-sdk-for-java
+Azure\azure-sdk-for-java-pr
+Azure\azure-sdk-for-python
+Azure\azure-sdk-for-python-pr
+Azure\azure-sdk-for-js
+Azure\azure-sdk-for-js-pr
+Azure\azure-sdk-for-c
+Azure\azure-sdk-for-c-pr
+Azure\azure-sdk-for-cpp
+Azure\azure-sdk-for-go
+Azure\azure-sdk-for-go-pr
```

Comment:
no android/ios?

Reply:
I can't speak to why, but it wasn't part of these efforts.

Reply:
I figured that was probably the case, just wanted to double check.